server/testdriver/junit5/src/main/java/org/infinispan/server/test/junit5/InfispanServerExtensionBuilder.java

```diff
@@ -0,0 +1,43 @@
+package org.infinispan.server.test.junit5;
+
+import org.infinispan.server.test.core.AbstractServerConfigBuilder;
+import org.infinispan.server.test.core.ServerRunMode;
+
+import static org.infinispan.server.test.core.AbstractInfinispanServerDriver.DEFAULT_CLUSTERED_INFINISPAN_CONFIG_FILE_NAME;
+
+/**
+ * Infinispan Server Extension Builder
+ *
+ * @author Katia Aresti
+ * @since 11
+ */
+public class InfispanServerExtensionBuilder extends AbstractServerConfigBuilder<InfispanServerExtensionBuilder> {
+   /**
+    * Use this method to instantiate a single clustered embedded server
+    *
+    * @return InfinispanServerExtension
+    */
+   public static InfinispanServerExtension server() {
+      return server(true);
+   }
+
+   public static InfinispanServerExtension server(boolean container) {
```

Comment:
Is the `container` parameter still needed?

We also need a test to make sure it works, there is no usage ATM.

Reply:
adding a test